### PR TITLE
feat: oval footer

### PR DESCRIPTION
**Pull request by gsteenkamp89**
## screenshots

Screenshot 2024-01-25 at 12 07 54

### CODE REVIEW

File: src/constant/links.tsx [hunk 1]
```diff
@@ -117,3 +117,30 @@ export const footerLinks = {
...
+      label: "For Searchers",
+      href: "https://docs.oval.xyz/for-searchers",
+    },
+    {
+      label: "For Integrators",
+      href: "https://docs.oval.xyz/integration",
+    },
+    {
+      label: "Stats",
+      href: "https://stats.uma.xyz/",
```
Comment (daywiss) [inline src/constant/links.tsx:133]: im wondering if this is hte right stats page to show, its based on mostly osnap and has no oval stats. perhaps one of these are better?
https://dune.com/risk_labs/aave-oev-via-liquidation-bonuses
or
https://dune.com/risk_labs/compound-v2-and-v3-liquidation-incentives

**Review by daywiss — COMMENTED** [review]
not sure if stats.uma is the correect link

File: src/constant/links.tsx (hunk 1 above)
Comment (daywiss) [reply src/constant/links.tsx:133]: actually might be better to just link to the docs: https://docs.oval.xyz/oev-data

Comment (gsteenkamp89) [reply src/constant/links.tsx:133]: oh this is a mistake. this is from the uma homepage's footer 😬

Comment (gsteenkamp89) [reply src/constant/links.tsx]: removing it entirely

**Review by daywiss — APPROVED** [review]
great

**Pull request merged**